secure-biometric-interface-integrator/lib/secureBiometricInterface.js

```diff
@@ -275,21 +275,18 @@ class SecureBiometricInterface {
 
   /**
    * Generate the dropdown menu list (a div which contain all dropdown option)
-   * @param {HTMLElement} [optionElement=null] dropdown options div
    * @returns HTMLElement dropdown menu list container
    */
-  generateDropdownMenuList(optionElement = null) {
-    if (optionElement == null) {
-      optionElement = this.generateOptionElement(this.modalityDevices);
-    }
+  generateDropdownMenuList() {
+    let optionElement = this.generateOptionElement(this.modalityDevices);
     const dropdownMenuList = this.container.querySelector(
       ".sbd-dropdown__menu-list"
     );
     if (dropdownMenuList) {
       dropdownMenuList.innerHTML = "";
       if (Array.isArray(optionElement)) {
         appendArray(dropdownMenuList, optionElement);
-      } else if (optionElement !== null) {
+      } else if (optionElement != null) {
```

Comment:
It should be !== ? Why did we change?

Reply:
hi @gsasikumar , we have remove the above piece of code with if condn, actually it is an unreachable code, i have added this due to some edge cases, may be those cases has been handle in other methods. So thats why remove this if condn code. Please check [6837e97](https://github.com/mosip/esignet-plugins/pull/38/commits/6837e97a1f27a790c1211de4409c2bc4e14b213d) commit